internal/generator/generator.go

```diff
@@ -8,18 +8,17 @@ import (
 	"html/template"
 	"log"
 
-	"github.com/Joffref/genz/internal/parser"
 	"github.com/Masterminds/sprig/v3"
 	"golang.org/x/tools/go/packages"
 )
 
-type parseFunc func(pkg *packages.Package, structName string) (parser.Struct, error)
+type ParseFunc func(pkg *packages.Package, structName string) (interface{}, error)
```

Comment:
Why wouldn't we keep working with parser.Struct ?
I see no issues having Attributes field empty

(Just rename struct maybe)